interface/wx/dirdlg.h

```diff
@@ -43,6 +48,13 @@ const char wxDirDialogNameStr[] = "wxDirCtrl";
     @style{wxDD_CHANGE_DIR}
            Change the current working directory to the directory chosen by the
            user.
+           @note This flag is ignored if the @c wxDD_MULTIPLE style is also used
+    @style{wxDD_MULTIPLE}
+           Allow the user to select multiple directories.
+           @since 3.1.4
```

Comment:
I have seen this kind of formatting issue before, but I think they aren't present on the doxygen version that is used to generate the actual docs (since they don't appear in the online docs). Perhaps @vadz can shed some light on this.

Reply:
The ones on the wxGauge documentation page you linked look correct though, since they don't cause issue with the future spacing. I would guess this is an issue that was created with later Doxygen versions, so I will go ahead and make it a sentence instead of a macro for better compatibility.

Reply:
I have removed the `@since` usage and made them a separate sentence.